docs/docsite/rst/installation_guide/intro_installation.rst

```diff
@@ -64,40 +61,33 @@ Managed Node Requirements
 
 On the managed nodes, you need a way to communicate, which is normally ssh. By
 default this uses sftp. If that's not available, you can switch to scp in
-:file:`ansible.cfg`.  You also need Python 2.6 or later.
-
-.. note::
-
-   Ansible's "raw" module (for executing commands in a quick and dirty
-   way) and the script module don't even need that.  So technically, you can use
-   Ansible to install python-simplejson using the raw module, which
-   then allows you to use everything else.  (That's jumping ahead
-   though.)
+:file:`ansible.cfg`.  You also need Python 2 (version 2.6 or later) or Python 3 (version 3.5 or
+later).
 
 .. note::
 
-   If you have SELinux enabled on remote nodes, you will also want to install
-   libselinux-python on them before using any copy/file/template related functions in
-   Ansible. You can of course still use the yum module in Ansible to install this package on
-   remote systems that do not have it.
+   * If you have SELinux enabled on remote nodes, you will also want to install
+     libselinux-python on them before using any copy/file/template related functions in Ansible. You
+     can use the :ref:`yum` or :ref`dnf` module in Ansible to install this package on remote systems
+     that do not have it.
 
-.. note::
-
-   Ansible 2.2 introduces a tech preview of support for Python 3. For more information, see `Python 3 Support <http://docs.ansible.com/ansible/python_3_support.html>`_.
+   * By default, Ansible uses the python interpreter located at :file:`/usr/bin/python` to run its
+     modules.  However, some Linux distributions may only have a Python 3 interpreter installed to
+     :file:`/usr/bin/python3` by default.  On those systems, you can either set the
```

Comment:
What error or behavior would the user see in this scenario?

Reply:
Fixed.  Latest version adds the error to look out for.